plugins/azure/frontdoor/frontDoorManagedIdentity.js

```diff
@@ -0,0 +1,60 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Front Door Managed Identity Enabled',
+    category: 'Front Door',
+    domain: 'Content Delivery',
+    severity: 'Medium',
+    description: 'Ensures that Azure Front Door have managed identity enabled.',
+    more_info: 'Enabling managed identities eliminate the need for developers having to manage credentials by providing an identity for the Azure resource in Azure AD and using it to obtain Azure Active Directory (Azure AD) tokens.',
+    recommended_action: 'Modify the Front Door classic profile and add managed identity.',
```

Comment:
If its only available for classic profiles we should also mention this in description